.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't mind if you use VS Code but personally I don't, and I think I'm happy to let people use whatever local configuration they wish without being prescriptive about it. You can add `.vscode` to the `.gitignore` maybe?

Reply:
Specifically re `extensions.json`: these pop up as "recommended extensions for this repo" to developers who open this repo. It's advisory, not prescriptive, and its only purpose is to be checked in.

The settings in `.vscode/settings.json` are also strictly the ones that conform to this repo's current formatting/linting scheme, codifying this repo's style + checkers, so that they're in a default-ready state…

But: it could be annoying to developers who want to e.g. toggle linting temporarily; so I agree that the `settings.json` could or should be removed or omitted. I looked for examples, and in TypeScript's repo, which they renamed as `settings.template.json` with a comment to self-manage: https://github.com/microsoft/TypeScript/tree/main/.vscode

So that's an option, too. Though, I don't feel strongly about these files; they're just developer assists and don't directly affect CI or builds.

Reply:
The other thing is that not every developer uses VS Code. Are you going to add similar files for JetBrains PyCharm? Eclipse? Where do you draw the line? I don't even use an IDE for this project.

Reply:
No — I don't use those either, and so obviously wouldn't contribute configs for them. As for where you draw the line, since out-of-box configs are just "nice-to-haves", as long as they aren't likely to go stale, I would normally draw the line at wherever "nice to have" stops. But it's not worth the hassle. I'll close this PR.